interaction_model/intent_schema.json

```diff
@@ -5,11 +5,11 @@
       "slots": [
         {
           "name": "movieName",
-          "type": "AMAZON.LITERAL"
```

Comment:
This works? I'm in their Skill Builder UI and I don't see an option for any of these without the AMAZON namespace.